dandi/publish/storage.py

```diff
@@ -2,11 +2,61 @@
 
 from django.conf import settings
 from django.core.files.storage import Storage, get_storage_class
+from django.db import models
 from minio_storage.policy import Policy
 from minio_storage.storage import MinioStorage, create_minio_client_from_settings
 from storages.backends.s3boto3 import S3Boto3Storage
 
 
+class CallableStorageFileField(models.FileField):
+    """A FileField backporting the Django 3.1 feature to allow a callable "storage" argument."""
```

Comment:
Are you planning on removing this when we bump to 3.1?

Reply:
Yes.